src/qibolab/instruments/zhinst/components/channel.py

```diff
@@ -0,0 +1,19 @@
+from dataclasses import dataclass
+
+from qibolab.components import Channel
```

Comment:
https://github.com/qiboteam/qibolab/pull/885#discussion_r1584845159

Even avoiding explicit `from . import ...` the message doesn't change.

Reply:
I don't understand what is the point of your comment, but it can be made relative import for sure.